types/components/ToggleButton.d.ts

```diff
@@ -9,10 +9,11 @@ export interface ToggleButtonProps {
   name?: string;
   checked?: boolean;
   disabled?: boolean;
+  focused?: boolean;
   onChange?: React.ChangeEventHandler<this>;
+  onFocus?: React.FocusEventHandler<this>;
   value: unknown;
   inputRef?: React.LegacyRef<this>;
```

Comment:
```suggestion
```

Reply:
I actually did not touch the `inputRef` because I'm not 100% sure I can scrap it. I did remove `innerRef` which was the one previously in use on the wrapper component.

Reply:
it's just cruft; somehow that prop definition got duplicated

Reply:
So should I remove the `inputRef` in the component as well?

Reply:
my bad; you're right. i misread